.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,7 +11,14 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Test case or steps to reproduce the behavior or.
+Test case or steps to reproduce the behavior or.  
```

Comment:
Remove `or`, not sure why it's there.

Reply:
Fixed.
[94f4335](https://github.com/FasterXML/jackson-module-kotlin/pull/637/commits/94f43354c1a7d930422c35a491bd0d02cac0ffd9)